composer.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "friendsofphp/number",
+    "description": "Adds a Number utility class",
```

It looks like a commit message 😉. Maybe something more like:

```suggestion
    "description": "Number utility for displaying human-friendly numeric data",
```

---

Perfect, thanks!